server/homebrew.api.js

```diff
@@ -59,14 +59,14 @@ const getBrew = (accessType)=>{
 		}
 
 		// If after all of that we still don't have a brew, throw an exception
-		if(!stub) {
+		if(!stub && !stubOnly) {
 			throw 'Brew not found in Homebrewery database or Google Drive';
 		}
```

Comment:
Can you help me understand this change?  If we can't find any record of the brew (no stub and nothing from Google Drive), then shouldn't we always throw here whether `stubOnly` is true or not?

Reply:
If there's no stub and it's only getting the stub that handles Google drive pre-stub updates

Reply:
Hm.... OK, that makes sense. After this PR I might go in and rearrange some logic further up in this function because something in the back of my mind thinks there is something weird going on but I can't put my finger on it.

Reply:
There might be 🤷 any extra eyes would be greatly appreciated. This is why I want to unit test it 😅

Reply:
Yes we definitely need better testing on this project....